shared-ui/components/example-button/SecondaryButton.styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { max } from '../../lib/responsive';
+import { colors } from '../../style/colors';
+
+const StyledSecondaryButton = styled.button`
+  color: ${colors.BUTTON_GREEN};
+  background-color: ${colors.BUTTON_DARK_GREEN};
+  border-color: ${colors.BUTTON_GREEN};
+  padding: 1em 1em;
```

Comment:
I could be wrong but I think this could simplify to padding: 1em

Reply:
yes it can be simplified